src/Mod/PartDesign/Gui/TaskHoleParameters.h

```diff
@@ -121,6 +120,11 @@ private Q_SLOTS:
         TaskHoleParameters * owner;
         PartDesign::Hole * hole;
     };
+    enum class HoleTypeIndex : int {

```

Comment:
I wonder if in that case making it `enum class` is worth. We use it explicitly as int anyway, so maybe `class` is not really wanted here.

Reply:
using class has more type-safety, and not using it is more brief, makes the code more legible, removed class as requested.